assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -545,6 +667,47 @@ const baseSelectors = {
 		// update.
 		return state.isAwaitingModulesRefresh;
 	} ),
+
+	/**
+	 * Checks if we can activate a module with a given slug.
+	 *
+	 * Returns `true` if the module can be activated.
+	 * Returns `false` if the module can not be activated.
+	 * Returns `undefined` if slug can not be found in state.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} slug  Module slug.
+	 * @return {(boolean|undefined)} Can activate module status; `undefined` if state is still loading or if no module with that slug exists.
+	 */
+	canActivateModule( state, slug ) {
+		const moduleRequirements = state.checkRequirementsResults[ slug ];
```

Comment:
It's not strictly necessary (just since we don't do that everywhere else), but since you've added an `invariant` for requiring `slug` in the `getCheckRequirementsError` selector, I'd suggest to do that here too.

Reply:
Addressed.